tests/test_release_transpiler_argument_types.py

```diff
@@ -0,0 +1,564 @@
+        (
+            [
+                "combine",
+                "--release-folder",
+                VALID_RELEASE_FOLDER,
+                "--release-base",
+                VALID_RELEASE_BASE,
+                "--override-mapping",
+                VALID_OVERRIDE_MAPPING_FILE,
+                "--py_coords",
+                "3.8,3.6",
+            ],
+            does_not_raise(),
+        ),
+        (
+            [
+                "combine",
+                "--release-folder",
+                VALID_RELEASE_FOLDER,
+                "--release-base",
+                VALID_RELEASE_BASE,
+                "--override-mapping",
+                VALID_OVERRIDE_MAPPING_FILE,
+            ],
+            does_not_raise(),
+        ),
+        (
+            [
+                "combine",
+                "--release-folder",
+                VALID_RELEASE_FOLDER,
+                "--release-base",
+                VALID_RELEASE_BASE,
+                "--override-mapping",
+                VALID_OVERRIDE_MAPPING_FILE,
+                "--py_coords",
+                "py3.8",
+            ],
+            pytest.raises(ARGUMENT_PARSING_ERROR),
+        ),
+        (
+            [
+                "combine",
+                "--release-folder",
+                VALID_RELEASE_FOLDER,
+                "--release-base",
+                VALID_RELEASE_BASE,
+                "--override-mapping",
+                VALID_OVERRIDE_MAPPING_FILE,
+                "--py_coords",
+                "test",
+            ],
+            pytest.raises(ARGUMENT_PARSING_ERROR),
+        ),
+        (
+            [
+                "combine",
+                "--release-folder",
+                VALID_RELEASE_FOLDER,
+                "--release-base",
+                VALID_RELEASE_BASE,
+                "--override-mapping",
+                VALID_OVERRIDE_MAPPING_FILE,
+                "--py_coords",
+                "'3.8, 3.6'",
+            ],
+            pytest.raises(ARGUMENT_PARSING_ERROR),
+        ),
+        (
+            [
+                "combine",
+                "--release-folder",
+                VALID_RELEASE_FOLDER,
+                "--release-base",
+                VALID_RELEASE_BASE,
+                "--override-mapping",
+                VALID_OVERRIDE_MAPPING_FILE,
+                "--py_coords",
+                "4.2",
+            ],
+            pytest.raises(ARGUMENT_PARSING_ERROR),
+        ),
+        (
+            [
+                "combine",
+                "--release-folder",
+                VALID_RELEASE_FOLDER,
+                "--release-base",
+                VALID_RELEASE_BASE,
+                "--override-mapping",
+                VALID_OVERRIDE_MAPPING_FILE,
+                "--py_coords",
+                "true",
+            ],
+            pytest.raises(ARGUMENT_PARSING_ERROR),
+        ),
+    ],
+)
+def test_combine_py_coords_type(args: list, expectation, monkeypatch):
+    print("args: " + " ".join(args))
```

Comment:
remove the print

Reply:
I got "sveiseblind" by looking at the same code for so long 👨‍🏭